src/Translator/src/FileDumper/WebpackTranslationsDumper.php

```diff
@@ -72,16 +73,16 @@ public function dump(MessageCatalogueInterface ...$catalogues): void
 
         $this->filesystem->dumpFile($this->dumpDir.'/index.js', $translationsJs);
         $this->filesystem->dumpFile($this->dumpDir.'/index.d.ts', $translationsTs);
-        $this->filesystem->dumpFile($this->dumpDir.'/configuration.js', sprintf(
-            "export const localeFallbacks = %s;\n",
-            json_encode($this->getLocaleFallbacks(...$catalogues), \JSON_THROW_ON_ERROR)
-        ));
-        $this->filesystem->dumpFile($this->dumpDir.'/configuration.d.ts', <<<'TS'
-import { LocaleType } from '@symfony/ux-translator';
-
-export declare const localeFallbacks: Record<LocaleType, LocaleType>;
-TS
-        );
+    }
+
+    public function setExcludedDomains(array $domains = []): void
+    {
+        $this->excludedDomains = $domains;
+    }
+
+    public function setDomains(array $domains): void
+    {
+        $this->domains = $domains;
     }
```

Comment:
Wondering if this should either
* be in an abstractDumper
* be in a new TranslationDumpProvider (find a better name^^)

Because there is a certain amount of duplicated code there